api/policies/CheckPermissions.js

```diff
@@ -33,14 +33,29 @@ module.exports = class CheckPermissionsPolicy extends Policy {
           if (action !== 'create' && permission[0].relation === 'owner') {
             if (action === 'access' || !req.params.id) {
               if (action === 'update' || action === 'destroy') {
-                return next({
-                  message: 'Update and Destroy are not yet supported with permissions, please do your request manually',
-                  statusCode: 400,
-                  code: 'E_UNSUPPORTED'
+                this.app.services.FootprintService.find(modelName, {}, {
+                  populate: [{
+                    model: this.app.orm.User,
+                    as: 'owners',
+                    required: true,
+                    where: {id: req.user.id}
+                  }]
+                }).then(items => {
+                  if (items.length > 1000) this.app.log.warning('Caution, bulk update/destroy for large model sets can be unperformant')
+                  const ids         = items.reduce((acc, val) => {
+                    acc.push(val.id)
+                    return acc
+                  }, [])
+                  req.query.where = {
```

Comment:
It works but as you said it needs to honor the params provided by the user.
I'm looking into the file you linked for viewing if there's somethings I can reuse, I don't think that copy-pasting code is a "smart" move, maybe I can try to use the trailpack-footprint exposed methods?